tree_rightview

```diff
@@ -0,0 +1,61 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+struct node
+{
+	int data;
+	struct node* left;
+	struct node* right;	
+};
+
+struct node* insertNode(int data)
+{
+	struct node* temp=(struct node*)malloc(sizeof(struct node));
```

Comment:
new operator exists in C++. Also, no need for `struct` here and anywhere else (with the function return values).